dbt/adapters/athena/utils.py

```diff
@@ -9,6 +11,19 @@ def clean_sql_comment(comment: str) -> str:
     return " ".join(line for line in split_and_strip if line)
 
 
+def stringify_table_parameter_value(value: Any) -> str:
+    """Convert any variable to string for Glue Table property."""
+    if isinstance(value, (dict, list)):
+        value_str: str = json.dumps(value)
```

Comment:
there could be some edge cases where the call to `json.dump` could fail...
did you consider to add a try/Exception with a fallback to str(value) with truncation?